src/program/lwaftr/query/tests/test_query_reconfigurable.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+SKIPPED_CODE=43
+
+if [[ -z "$SNABB_PCI0" ]]; then
+    echo "SNABB_PCI0 not set"
+    exit $SKIPPED_CODE
+fi
+
+if [[ -z "$SNABB_PCI1" ]]; then
+    echo "SNABB_PCI1 not set"
+    exit $SKIPPED_CODE
+fi
+
+source ./program/lwaftr/query/tests/test_env.sh
+
+trap cleanup EXIT HUP INT QUIT TERM
+
+LWAFTR_CONF=./program/lwaftr/tests/data/no_icmp.conf
+LWAFTR_NAME=lwaftr-$$
+
+# Run lwAFTR.
+tmux_launch "lwaftr" "./snabb lwaftr run --reconfigurable --name $LWAFTR_NAME --conf $LWAFTR_CONF --v4 $SNABB_PCI0 --v6 $SNABB_PCI1" "lwaftr.log"
+sleep 2
+
+# Test query all.
+test_lwaftr_query -l
+
+# Test query by pid.
+pid=$(get_lwaftr_follower)
```

Comment:
I'd like to see tests that use the leader pid as well (similar to the tests using the follower pid) because that's the most intuitive way for end users to use our software.